test/common/directives/ng_template_outlet_test.dart

```diff
@@ -13,22 +13,25 @@ main() {
     test("should do nothing if templateRef is null", () async {
```

Comment:
I'm seeing 4 tests added to the existing 4 – total of 8.

Yay for tests!

But when I run, I see 7 of 8 failing.

Look into fixing these?